verify-core/src/main/java/io/mosip/verifycore/dto/authorizationRequest/AuthorizationRequestCreateDto.java

```diff
@@ -0,0 +1,17 @@
+package io.mosip.verifycore.dto.authorizationRequest;
+
+import io.mosip.verifycore.dto.presentation.PresentationDefinitionDto;
+import lombok.*;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+public class AuthorizationRequestCreateDto {
```

Comment:
Updated

Reply:
The following I felt it is little mismatching.
1. /vp-definition - PresentationDefinitionDto - PresentationDefinitionController - PresentationDefinitionService - This sequence is little mismatching
2. vc-verification - VerificationStatusDto - CredentialVerificationController - CredentialVerificationService
3. vp-request/{requestId}/status - this can be VPRequestStatusDto or VP RequestStatusResponseDto
4. /vp-direct-post - ResponseAcknowledgementDto - VPSubmissionController - VerifiablePresentationSubmissionService

Reply:
Point 1 to 3 is fine, on point 4, I think we discussed to keep the result part as different controller right ? because of the exposure to internet vs protected endpoint